common/api/appui-react.api.md

```diff
@@ -4178,6 +4218,28 @@ export class OpenMessageCenterEvent extends UiEvent<{}> {
 // @internal
 export function packNineZoneState(state: NineZoneState): SavedNineZoneState;
 
+// @beta
+export interface PanelConfig {
```

Comment:
@raplemie, @NancyMcCallB what do you think about the naming? Should this be a `StagePanelConfig` instead for consistency? I.e. we have a `StagePanelDef`. Ideally, I'd prefer to drop the `Stage` part/prefix..

Reply:
I wonder if panel wouldnt be too generic, maybe `SidePanelConfig` ? (But, it may not be necessary to change at all)

Reply:
I have renamed `PanelConfig` to `StagePanelConfig` for consistency, since we have a bunch of panel related types prefixed with `StagePanel`, i.e. `StagePanelLocation`, `StagePanelSection`...